tests/unit/style/properties/serialization.rs

```diff
@@ -1139,3 +1148,26 @@ mod shorthand_serialization {
 
     }
 }
+
+#[test]
+fn test_border_spacing() {
+    use cssparser::Parser;
+    use media_queries::CSSErrorReporterTest;
+    use style::parser::ParserContext;
+    use style::properties::longhands::border_spacing;
+    use style::stylesheets::Origin;
+
+    macro_rules! parse_assert {
```

Comment:
We can use this macro in the future for other longhands too. Could you move it to `parsing/mod.rs` and rename it something like `assert_parser_exhausted`. You can look at the `parse_longhand` macro for an example of this.